### PR TITLE
Clear Format On Exit Quote

### PR DESCRIPTION
**Pull request by BryanValverdeU**
Clear the format when user press ENTER on an empty line of a BLOCKQUOTE.

### CODE REVIEW

File: packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/quoteFeatures.ts [hunk 1]
```diff
@@ -74,16 +76,36 @@ function splitQuote(event: PluginKeyboardEvent, editor: IEditor) {
...
+        element.style.borderColor &&
+        element.style.paddingLeft &&
+        element.style.color
+    ) {
+        return true;
+    }
+    return false;
+};
+
 /**
```
Comment (BryanValverdeU) [inline packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/quoteFeatures.ts:109]: Added this check, so we verify that the QuoteElement is styled and it is not a blockquote used in indentation

**Comment by JiuqingSong** [comment]
What is the original bug you are fixing? And what it broken without this fix?

**Comment by BryanValverdeU** [comment]
> What is the original bug you are fixing? And what it broken without this fix?

This is the Bug: https://outlookweb.visualstudio.com/Outlook%20Web/_workitems/edit/125548
It seems that when exiting a Quote did not default format the text back to the default styles.

**Comment by JiuqingSong** [comment]
> > What is the original bug you are fixing? And what it broken without this fix?
> 
> This is the Bug: https://outlookweb.visualstudio.com/Outlook%20Web/_workitems/edit/125548
> It seems that when exiting a Quote did not default format the text back to the default styles.

Can you repro the original bug?

**Comment by BryanValverdeU** [comment]
Yes, I was able to repro, when the format of a quote text style is different: this is a snapshot.
`<blockquote style="border-left: 3px solid rgb(200, 200, 200); border-top-color: rgb(200, 200, 200); border-right-color: rgb(200, 200, 200); border-bottom-color: rgb(200, 200, 200); padding-left: 10px; color: rgb(102, 102, 102);"><div><span style="font-size: 36pt; color: rgb(243, 144, 29); font-family: Tahoma;">This is a quote</span></div></blockquote><div><span style="font-size: 36pt; color: rgb(243, 144, 29); font-family: Tahoma;">Text after exiting a quote with enter key</span></div><div>Text with Format Cleaned</div><!--{"start":[1,0,0,41],"end":[1,0,0,41]}-->`

**Review by JiuqingSong — APPROVED** [review]

**Pull request merged**